src/matrixrtc/MatrixRTCSession.ts

```diff
@@ -294,44 +284,57 @@ export class MatrixRTCSession extends TypedEventEmitter<
      *
      * It can be undefined since the callId is only known once the first membership joins.
      * The callId is the property that, per definition, groups memberships into one call.
+     *
+     * This may also be undefined if MSC4143 slots are in use, as calls do not always have an ID if they
+     * have a slot.
      * @deprecated use `slotId` instead.
      */
     public get callId(): string | undefined {
         return this.slotDescription?.id;
     }
-    /**
-     * The slotId of the call.
-     * `{application}#{appSpecificId}`
-     * It can be undefined since the slotId is only known once the first membership joins.
-     * The slotId is the property that, per definition, groups memberships into one call.
-     */
-    public get slotId(): string | undefined {
-        return slotDescriptionToId(this.slotDescription);
-    }
 
     /**
-     * Returns all the call memberships for a room that match the provided `sessionDescription`,
-     * oldest first.
-     *
-     * @deprecated Use `MatrixRTCSession.sessionMembershipsForSlot` instead.
+     * Get a slot for a given room and description
+     * @param room The room which the slot is scoped to.
+     * @param slotDescription The description of the slot. The type, ID, and parameters must exactly match.
+     * @returns The contents of the slot event, or null if no matching slot found.
      */
-    public static callMembershipsForRoom(
-        room: Pick<Room, "getLiveTimeline" | "roomId" | "hasMembershipState" | "_unstable_getStickyEvents">,
-    ): CallMembership[] {
-        return MatrixRTCSession.sessionMembershipsForSlot(room, {
-            id: "",
-            application: "m.call",
-        });
-    }
+    public static getRtcSlot(
+        room: Pick<Room, "getLiveTimeline" | "roomId">,
```

Comment:
oh that's new, we need the timelime now. I guess it is the only to access state?